query/security/common/src/main/java/org/codice/ddf/admin/security/common/fields/wcpm/ContextPolicyBin.java

```diff
@@ -146,4 +142,14 @@ public void initializeFields() {
         // TODO: 4/18/17 phuffer -  Replace with a MapField
         claimsMapping = new ListFieldImpl<>("claimsMapping", ClaimsMapEntry.class);
     }
+
+    public ContextPolicyBin useDefaultRequiredFields() {
+        isRequired(true);
+        contexts.isRequired(true);
+        authTypes.isRequired(true);
+        authTypes.getListFieldType().isRequired(true);
+        realm.isRequired(true);
+        claimsMapping.isRequired(false);
```

Comment:
Should set the Claim & ClaimValue inside of the ClaimsMapping object to true either here or with a similar `useDefaultRequiredFields` method inside the `ClaimsMapEntry` class.